.infrastructure/busybox.yml

```diff
@@ -2,11 +2,9 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: busybox
-  namespace: todoapp
+  namespace: mateapp
 spec:
   containers:
-  - name: busybox
-    image: ikulyk404/busyboxplus:curl
-    args:
-    - sleep
-    - "1000"
+    - name: busybox
+      image: niklafinskiy/alpine-curl:1.0.0
```

Comment:
The image specified here is `niklafinskiy/alpine-curl:1.0.0`, but the task requires using `busyboxplus:curl`. Please update the image to `busyboxplus:curl` to meet the task requirements.